lib/gst/plugin/gstgtkutils.c

```diff
@@ -146,7 +146,10 @@ gst_gtk_get_width_height_for_rotation (gint width, gint height,
     gint *out_width, gint *out_height,
     GstVideoOrientationMethod rotation)
 {
-  if (rotation == GST_VIDEO_ORIENTATION_90R || rotation == GST_VIDEO_ORIENTATION_90L) {
+  if (rotation == GST_VIDEO_ORIENTATION_90R ||
+      rotation == GST_VIDEO_ORIENTATION_90L ||
+      rotation == GST_VIDEO_ORIENTATION_UL_LR ||
+      rotation == GST_VIDEO_ORIENTATION_UR_LL) {
```

Comment:
nitpick: With so many, a switch-case might be cleaner here for setting `out_width` and `out_height`.